lib/fuse.c

```diff
@@ -1075,26 +1073,30 @@ static int try_get_path(struct fuse *f, fuse_ino_t nodeid, const char *name,
 	return err;
 }
 
-static void queue_element_unlock(struct fuse *f, struct lock_queue_element *qe)
+static int try_get_path2(struct fuse *f, fuse_ino_t nodeid1, const char *name1,
```

Comment:
Could you move definition of this function to where it originally was (further down) to simplify the diff?

Reply:
Only by forward declaring it, and forward declaring (and taking the permanent complexity hit, however slight) just to simplify the diff during review feels weird to me.

I split out the movement of this function to an earlier commit, though I know that github's UI doesn't make that all that obvious. The second commit in the stack should be just the actual changes, if you want to review the two commits separately. I can also send a separate PR instead of a multi-commit PR if that would help.

Reply:
For me it is ok, I see it as two different patches. I just really didn't find the time to review, as I wrote before, will definitely do this week.